packages/calcite-components/src/assets/styles/_sortable.scss

```diff
@@ -1,8 +1,24 @@
 @mixin sortable-helper-classes() {
-  .calcite-sortable--chosen,
-  .calcite-sortable--ghost,
+  .calcite-sortable--ghost::before {
+    content: "";
+    @apply box-border
+      border
+      border-dashed
+      border-color-brand
+      bg-foreground-2
+      absolute
+      top-0
+      left-0
+      bottom-0
+      right-0
+      z-default;
+  }
   .calcite-sortable--drag {
-    @apply border border-dashed;
-    border-color: var(--calcite-ui-brand);
+    @apply shadow-2;
+  }
+  .calcite-sortable--original-location {
```

Comment:
Does `calcite-sortable--original-location` need to be defined [here](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/utils/sortableComponent.ts#L13-L15) in this PR? Also, Is dropping the `chosen` class intentional?

Reply:
`original-location` isn't needed. I forgot to remove. Will do that now.

Removing `chosen` class is intentional since the Figma designs show no styling changes for when something is chosen vs dragged.